app/templates/devresources/.editorconfig

```diff
@@ -0,0 +1,18 @@
+# see http://editorconfig.org
```

Comment:
is this required ?

Reply:
Yes. This helps with consistent code styling across editors.